src/core/fonts.js

```diff
@@ -722,6 +722,11 @@ var Font = (function FontClosure() {
   function int16(b0, b1) {
     return (b0 << 8) + b1;
   }
+  
+  function signed_int16(b0,b1) {
```

Comment:
Nit: missing space after the comma, it should be `function signed_int16(b0, b1) {`